README.md

```diff
@@ -1 +1,49 @@
 # EdgeTier-ML-Challenge
+
+Welcome to the EdgeTier machine learning and data science code challenge!
+
+You are required to read the challenge below, and submit a Jupyter notebook (or similar) with your analysis and output in answer to each part of the challenge.
+
+# Challenge
+
+SomeAnalytics Company are a fictional company who specialise in analysing, and classifying social media data related to a particular topic and sell insights on to businesses who may be interested.
+
+You have been hired by SomeAnalytics to analyse 500,000 tweets related to ChatGPT between January and March 2023 and help them with the following three asks they are seeing demand for among their clients:
+
+### Key Players and hashtags
+
+SomeCompany want you to identify who are the most 10 mentioned Twitter accounts in the dataset and what are the 20 most used hashtags in the dataset.
+
+The output here should be a table containing the Twitter handle or hashtag along with the number and % of tweets they appear in.
```

Comment:
Should we ask for a visualisation too? Like even a bar chart - nice check to see if they label axes / size it correctly / have an eye for legend/colour?